attic/deployments/dev/environment

```diff
@@ -30,6 +30,8 @@ _set AZUL_DEBUG 1
 
 _set AZUL_OWNER hannes@ucsc.edu
 
+_set AZUL_AWS_ACCOUNT_ID 122796619775
```

Comment:
The attic is for "deleted" files. We don't need to, and should not, modify them.